bazel/get_workspace_status

```diff
@@ -23,10 +23,24 @@ then
 fi
 
 # The code below presents an implementation that works for git repository
-git_rev=$(git rev-parse HEAD)
+git_rev=$(git rev-parse HEAD 2>&1)
 if [[ $? != 0 ]];
 then
-    exit 1
+    # However, if this is in fact a non-git and the SOURCE_VERSION does not
+    # exist: grab ${version} from DEPRECATED.md.
+    version=`cat DEPRECATED.md | grep "## Version" | head -n 1 | cut -d' ' -f3`
```

Comment:
@dio can we potentially wait on this until @htuch moves data-plane-api into this repo? Then we will have access to the VERSION file, and can potentially use that as a fallback?

Reply:
Yes, that will be better for sure. Should we close this first?

Reply:
I will let you coordinate with @htuch. I think he is going to do that work soon (this week) so probably OK to leave open to track, but either way.